pkg/cmd/open.go

```diff
@@ -45,7 +45,7 @@ var openStatusCmd = &cobra.Command{
 	Short: "open the Doppler status page",
 	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
-		err := open.Run("https://status.doppler.com")
+		err := open.Run("https://www.dopplerstatus.com")
```

Comment:
```suggestion
		err := open.Run("https://dopplerstatus.com")
```

Reply:
It 301 redirects to `www`, might as well save users the redirect.